packages/client/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@slicemachine/client",
+  "version": "1.1.10-alpha.0",
+  "private": false,
+  "main": "./build/index.js",
+  "engines": {
+    "node": ">=12"
+  },
+  "types": "./build/index.d.ts",
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc --project tsconfig.json",
+    "watch": "tsc --project tsconfig.json --watch",
+    "lint": "eslint --cache --cache-location .caches/eslint --cache-strategy content --ext .ts .",
+    "lint:precommit": "eslint",
+    "audit": "better-npm-audit audit -l high -p",
+    "prepublishOnly": "npm run test && npm run build",
+    "audit-fix": "npm audit fix",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@prismicio/types-internal": "^0.2.8",
+    "@slicemachine/core": "1.1.10-alpha.0",
+    "axios": "^0.27.2",
+    "mime": "^2.5.2",
+    "fp-ts": "^2.11.5",
+    "io-ts": "^2.2.16"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.15.8",
+    "@babel/preset-env": "^7.15.8",
+    "@babel/preset-typescript": "^7.15.0",
+    "@types/axios": "^0.14.0",
+    "@types/jest": "^27.0.2",
+    "@types/mime": "^2.0.3",
+    "@types/node": "^16.10.5",
+    "@typescript-eslint/eslint-plugin": "^5.10.0",
+    "@typescript-eslint/parser": "^5.10.0",
+    "better-npm-audit": "^3.4.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-import-resolver-typescript": "^2.4.0",
+    "eslint-plugin-import": "^2.24.0",
+    "eslint-plugin-jest": "^24.4.0",
+    "jest": "^27.2.4",
+    "nock": "^13.1.3",
+    "ts-jest": "^27.0.5",
+    "types-package-json": "^2.0.39",
+    "typescript": "^4.4.3",
+    "unionfs": "^4.4.0"
```

Comment:
tbh I would recommend using `mock-fs` over unionfs+memfs it's a lot easier to use, and less buggy :) 
https://www.npmjs.com/package/mock-fs

Reply:
I use nock in the end so that just remnant from the Core haha.
I did a depcheck on Core / Init / Client and removed some stuff 😄